CMakeLists.txt

```diff
@@ -411,8 +411,11 @@ if(WITH_CORE)
     message(STATUS "Qt WebKit support DISABLED.")
   endif()
 
+  set (WITH_INTERNAL_LAZPERF TRUE CACHE BOOL "Determines whether LazPerf should be built from internal copy")
```

Comment:
Shouldn't this be set to false by default or is the internal one generally preferable?

Reply:
The internal copy is generally preferable. It would be a rare case that someone would want to use external lazperf. In PDAL >= 2.4 the lazperf library is vendored and external lazperf is not supported anymore (AFAIK)

Reply:
Should this be an advanced (dev) option then?
Currently it looks tempting for packager to use distribution libs.

Reply:
I don't have a strong opinion on that.

WITH_INTERNAL_MDAL and WITH_INTERNAL_POLY2TRI are also true by default, and they're not marked as advanced...

Reply:
I think we could be doing better in helping packagers understand if we ship a lib internally because it's better (lazperf) or for convenience (qwtpolar)
That being said, happy to have this merged :-)

Reply:
Thanks @m-kuhn @wonder-sk 

+1 for better explanation. I'll prepare a PR for that purpose.